migrations/v2.js

```diff
@@ -0,0 +1,139 @@
+import {
+  describe,
+  whereFromPlugin,
+  whereContent,
+  mutateContent,
+  checkContent,
+  updatePlugin,
+  getConfig
+} from 'adapt-migrations';
+import _ from 'lodash';
+
+function getSpoorConfig() {
+  return getConfig()?._spoor;
+}
+
+/**
+ * `_tracking._shouldStoreResponse` default set to false but task updated to use v3.0.0 value
+ * the following attributes were missing default values - migration not included as not possible to discern author values from defaults:
+ * _isEnabled
+ * _tracking: _requireCourseCompleted, _requireAssessmentPassed, _shouldSubmitScore
+ * _reporting: _onTrackingCriteriaMet, _onAssessmentFailure
+ * _advancedSettings: _showDebugWindow, _commitOnStatusChange, _timedCommitFrequency, _maxCommitRetries, _commitRetryDelay
+ */
+describe('adapt-contrib-spoor - v2.0.0 to v2.0.2', async () => {
+  let spoorConfig;
+  const shouldStoreResponsesPath = '_tracking._shouldStoreResponses';
+  const scormVersionPath = '_advancedSettings._scormVersion';
+  whereFromPlugin('adapt-contrib-spoor - from v2.0.0 to v2.0.2', { name: 'adapt-contrib-spoor', version: '<2.0.2' });
+  whereContent('adapt-contrib-spoor - where _spoor', async () => {
+    spoorConfig = getSpoorConfig();
+    return spoorConfig;
+  });
+  mutateContent('adapt-contrib-spoor - add _spoor._tracking._shouldStoreResponses', async () => {
+    if (!_.has(spoorConfig, shouldStoreResponsesPath)) _.set(spoorConfig, shouldStoreResponsesPath, true);
+    return true;
+  });
+  checkContent('adapt-contrib-spoor - check _spoor._tracking._shouldStoreResponses added', async () => {
+    const isValid = _.has(spoorConfig, shouldStoreResponsesPath);
+    if (!isValid) throw new Error(`_spoor.${shouldStoreResponsesPath} not added`);
+    return true;
+  });
+  mutateContent('adapt-contrib-spoor - remove _spoor._advancedSettings._scormVersion', async () => {
+    _.unset(spoorConfig, scormVersionPath);
+    return true;
+  });
+  checkContent('adapt-contrib-spoor - check _spoor._advancedSettings._scormVersion removed', async () => {
+    const isValid = !_.has(spoorConfig, scormVersionPath);
+    if (!isValid) throw new Error(`_spoor.${scormVersionPath} not removed`);
+    return true;
+  });
+  updatePlugin('adapt-contrib-spoor - update to v2.0.2', { name: 'adapt-contrib-spoor', version: '2.0.2', framework: '>=2.0.0' });
+});
+
+describe('adapt-contrib-spoor - v2.0.0 to v2.0.5', async () => {
```

Comment:
The majority of the describes/whereFromPlugin are from v2.0.0, to be consistent with others can we just update these to the version before the one we're upgrading to. 
```suggestion
describe('adapt-contrib-spoor - v2.0.4 to v2.0.5', async () => {
```